cli/azd/pkg/tools/azcli/cognitive_service.go

```diff
@@ -0,0 +1,73 @@
+package azcli
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/Azure/azure-sdk-for-go/sdk/resourcemanager/cognitiveservices/armcognitiveservices"
+)
+
+func (cli *azCli) GetCognitiveAccount(
+	ctx context.Context,
+	subscriptionId string,
+	resourceGroupName string,
+	accountName string) (armcognitiveservices.AccountsClientGetResponse, error) {
```

Comment:
We probably should return `armcognitiveservices.Account` instead of `armcognitiveservices.AccountsClientGetResponse`, which seems more appropriate.

The SDK usually has the response wrapping the response object. Relevant snippet:
```golang
// AccountsClientGetResponse contains the response from method AccountsClient.Get.
type AccountsClientGetResponse struct {
	Account
}
```

Reply:
makes sense